### PR TITLE
chore: update promo video

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- chore: update promo video

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

**Review by bryanoltman — APPROVED** [review]

**Comment by codesculpture** [comment]
This video can be uploaded somewhere than github and just consumed here, i understand this is something not really important. But just saying.🙂

**Comment by felangel** [comment]
> This video can be uploaded somewhere than github and just consumed here, i understand this is something not really important. But just saying.🙂

What would be the benefit? It’s nice to keep in in the same place as the rest of the website source imo 🤷‍♂️

**Comment by codesculpture** [comment]
Github can only make process diff on structured text files such as our code, and it cannot able to process the raw video and image, imo these raw assets are should be hosted at some cdn or public hosting. I just believe github is only for code of any structured data that can able to diff the changes across commit. I mean its not a big deal, but as more assets came, the commit history could have lot of changes which is for these assets which seems uneccesary.


In summary, github history dont need to contain these assets changes. Moving these assets in somewhere and just having the url here would be nice. You can take this as a suggestion.
